package.json

```diff
@@ -57,7 +57,7 @@
     "passport-twitter": "^1.0.2",
     "phantomjs": ">=1.9.0",
     "serve-favicon": "^2.3.0",
-    "socket.io": "^1.3.5",
+    "socket.io": "Automattic/socket.io",
```

Comment:
I'm a little bit concerned and maybe we should wait for the next socket.io version. Unclear when it's released tho.

This line equals `"socket.io": "socketio/socket.io",`